Makefile

```diff
@@ -8,11 +8,16 @@ HELIO_OPENSSL_USE_STATIC_LIBS = ON
 HELIO_ENABLE_GIT_VERSION = ON
 HELIO_WITH_UNWIND = OFF
 
+# equivalent to: if $(uname_m) == x86_64 || $(uname_m) == amd64
+ifneq (, $(filter $(BUILD_ARCH),x86_64 amd64))
```

Comment:
Also isn't this only for the architecture and for the processor family ? I would expect something here that looks the microprocessor architecture and its familly

Reply:
no, we support x86_64 and arm64 architectures and we release binaries that can run for both architectures. 
See here: https://github.com/dragonflydb/dragonfly/releases

Reply:
I know which architectures we support, it appeared to me that this PR was meant to address the microarchitecture mentioned on that PR. I got confused, that's all